### DISCUSSION TITLE
webstorm和vscode均没有组件代码提示。。。

### DISCUSSION THREAD

**Discussion by LainNetWork** [jdf2e/nutui#960] [Q&A]
<img width="454" alt="image" src="https://user-images.githubusercontent.com/20994069/150165358-375993cc-c788-409c-a309-2bef92efef38.png">
拉了官方的demo项目下来也一样
我这边是前端萌新，找了一晚上原因了。。不知道是为什么T T，有没有大佬愿意指点一二

**Comment by richard1015** [comment]
https://nutui.jd.com/#/zh-CN/guide/ide
